matrix_client/api.py

```diff
@@ -180,11 +179,24 @@ def send_message_event(self, room_id, event_type, content, txn_id=None):
         self.txn_id = self.txn_id + 1
 
         path = ("/rooms/%s/send/%s/%s" %
-            (urllib.quote(room_id), urllib.quote(event_type),
-             urllib.quote(unicode(txn_id)))
+            (quote(room_id), quote(event_type), quote(str(txn_id)))
         )
         return self._send("PUT", path, content)
 
+    # content_type can be a image,audio or video
+    # extra information should be supplied, see https://matrix.org/docs/spec/r0.0.1/client_server.html
+    def send_content(self, room_id, item_url, item_name, item_type, extra_information=None):
+        if extra_information == None:
+            extra_information = {}
+
+        content_pack = {
+            "url":item_url,
+            "msgtype":"m."+item_type,
```

Comment:
Please can we drop the `m.` auto-filling and rely on the caller to give the right `msgtype`. They need to know the API fields anyway, so transforming `msgtype` into `item_type` doesn't really make anything clearer, and makes it impossible for people to use non-standard `msgtypes` (e.g. of the form `org.matrix.custom.msgtype`).